walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -27,11 +27,11 @@
     List<TimeRange> initalRanges = new ArrayList<TimeRange>();
     List<TimeRange> results = new ArrayList<TimeRange>();
     
-    for(int start = 0; start <= TimeRange.END_OF_DAY; start += 30) {
-        if(start + 30 > TimeRange.END_OF_DAY) {
+    for(int start = 0; start <= TimeRange.END_OF_DAY; start += 15) {
```

Comment:
nit: Since "15" is used in a few places here, consider pulling it into a named const variable. The idea is it works as inline documentation, as well as giving you a single point to update.

To be clear: something like: "minMeetingTime", not "fifteenMinutes" :)